noinst/specs/analyze_specs.R

```diff
@@ -0,0 +1,154 @@
+# This is a scratchpad for
```

Comment:
Do we need this file? If not, could you either delete of add some comments on top so we know what it is good for?

Reply:
No I think it can be deleted, especially now that the bulk of it is also in `specs_from_pdf`.
I mainly kept it to illustrate what else can be done with the date from `analyze_output`.